templates/contribute/doc.md

```diff
@@ -192,50 +158,6 @@ to preview the final result. See also the math.stackexchange
 [MathJax tutorial](https://math.meta.stackexchange.com/questions/5020/mathjax-basic-tutorial-and-quick-reference).
 
 
-## Tactic doc entries
```

Comment:
I created an issue at https://github.com/leanprover-community/leanprover-community.github.io/issues/319 to track the fate of this section.